components/menuData.ts

```diff
@@ -628,6 +628,7 @@ menuDataRouter.get(
                     controller.challengeService.getChallengesForContract(
                         getSession(req.jwt.unique_name).contractId,
                         req.gameVersion,
+                        // TODO: Should a difficulty be passed here?
```

Comment:
Probably? The game might handle it though (I doubt this).

Reply:
What is `/missionrewards` used for?